doc/UI_TESTS.md

```diff
@@ -82,18 +80,41 @@ def is_displayed(self):
 Method check if elements are present on page and path is in current url.
 
 
-####Widgets
-Every page object is composed from widgets.
+### Widgets
+Every page object is composed of widgets.
 Most components can be used from widgetastic libraries, 
 there you can find examples and usage(using selectors, actions etc.)
 But there are also 3scale specific widgets/elements which have to be implemented separately or can be
 inherited and changed.
 
 3scale specified widgets are located in `testsuite/ui/widgets/__init__.py`
 
-####Navigation
+### Navigation
+
+Navigator takes responsibility of navigation process during UI testing.
+It uses two basic structures (prerequisites and steps) in order to construct logical path (or sequence of actions)
+from root View to desired location. This simulates user actions during system navigation.
+
+Navigation process consists of two parts:
+   * 1. **Backtrace** - Views should extend NavigateStep class which defines simple method `prerequisite`.
+        This method should return/specify View that specific is ancestor of accessible from(is navigable from).
```

Comment:
Sentence "This method should return/specify View that specific is ancestor of accessible from(is navigable from)" doesn't seem clear to me.